HW3(REST)/event-service-impl/src/main/java/com/example/eventserviceimpl/Entity/EventEntity.java

```diff
@@ -0,0 +1,25 @@
+package com.example.eventserviceimpl.Entity;
+
+import lombok.*;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class EventEntity {
```

Comment:
[info] will be better to provide an table name and column names explicitly - it helps with navigation and validation of schema when you're using tools like IntelliJ IDEA ultimate.

(no changes required)